@rjsparks actually, I just tried `PersonFactory(user=None)` and PersonFactory isn't happy at all because it assumes the existence of a user, so you get several stacktraces like:

```
Traceback (most recent call last):
  File "/workspace/ietf/ietfauth/tests.py", line 534, in test_reset_password_without_user
    person = PersonFactory(user=None)
  File "/home/dev/.local/lib/python3.9/site-packages/factory/base.py", line 40, in __call__
    return cls.create(**kwargs)
  File "/home/dev/.local/lib/python3.9/site-packages/factory/base.py", line 528, in create
    return cls._generate(enums.CREATE_STRATEGY, kwargs)
  File "/home/dev/.local/lib/python3.9/site-packages/factory/django.py", line 121, in _generate
    return super()._generate(strategy, params)
  File "/home/dev/.local/lib/python3.9/site-packages/factory/base.py", line 465, in _generate
    return step.build()
  File "/home/dev/.local/lib/python3.9/site-packages/factory/builder.py", line 270, in build
    step.resolve(pre)
  File "/home/dev/.local/lib/python3.9/site-packages/factory/builder.py", line 211, in resolve
    self.attributes[field_name] = getattr(self.stub, field_name)
  File "/home/dev/.local/lib/python3.9/site-packages/factory/builder.py", line 356, in __getattr__
    value = value.evaluate_pre(
  File "/home/dev/.local/lib/python3.9/site-packages/factory/declarations.py", line 67, in evaluate_pre
    return self.evaluate(instance, step, context)
  File "/home/dev/.local/lib/python3.9/site-packages/factory/declarations.py", line 119, in evaluate
    return self.function(instance)
  File "/workspace/ietf/person/factories.py", line 71, in <lambda>
    name = factory.LazyAttribute(lambda p: normalize_name('%s %s'%(p.user.first_name, p.user.last_name)))
AttributeError: 'NoneType' object has no attribute 'first_name'
```

--- a/ietf/ietfauth/tests.py
+++ b/ietf/ietfauth/tests.py
@@ -527,6 +527,24 @@ class IetfAuthTests(TestCase):
         self.assertIn(secondary_address, to)
         self.assertNotIn(inactive_secondary_address, to)
 
+    def test_reset_password_without_user(self):
+        """Reset password using email address for person without a user account"""
+        url = urlreverse('ietf.ietfauth.views.password_reset')
+        email = EmailFactory()
+        person = email.person
+        # Remove the user object from the person to get a Email/Person without User:
+        person.user = None
+        person.save()
+        # Remove the remaining User record, since reset_password looks for that by username:
+        User.objects.filter(username__iexact=email.address).delete()
+        empty_outbox()
+        r = self.client.post(url, { 'username': email.address })
+        self.assertEqual(len(outbox), 1)
+        lastReceivedEmail = outbox[-1]
+        self.assertIn(email.address, lastReceivedEmail.get('To'))
+        self.assertTrue(lastReceivedEmail.get('Subject').startswith('Confirm registration at'))
+        self.assertContains(r, 'We have sent you an email with instructions', status_code=200)
+
     def test_review_overview(self):
         review_req = ReviewRequestFactory()
         assignment = ReviewAssignmentFactory(review_request=review_req,reviewer=EmailFactory(person__user__username='reviewer'))

--- a/ietf/ietfauth/views.py
+++ b/ietf/ietfauth/views.py
@@ -491,8 +491,11 @@ def password_reset(request):
             if not user:
                 # try to find user ID from the email address
                 email = Email.objects.filter(address=submitted_username).first()
-                if email and email.person and email.person.user:
-                    user = email.person.user
+                if email and email.person:
+                    if email.person.user:
+                        user = email.person.user
+                    else: 
+                        send_account_creation_email(request, email.email_address())
 
             if user and user.person.email_set.filter(active=True).exists():
                 data = {

